pkg/api/v1/schema.go

```diff
@@ -158,37 +145,17 @@ type MAC struct {
 	MAC string `json:"address"`
 }
 
-type FilterRef struct {
-	Filter string `json:"filter"`
-}
-
-type InterfaceSource struct {
-	Network string `json:"network,omitempty"`
-	Device  string `json:"device,omitempty"`
-	Bridge  string `json:"bridge,omitempty"`
-}
-
 type Model struct {
 	Type string `json:"type"`
 }
 
-type InterfaceTarget struct {
-	Device string `json:"dev"`
-}
-
-type Alias struct {
-	Name string `json:"name"`
-}
-
 // END Inteface -----------------------------
 //BEGIN OS --------------------
 
 type OS struct {
-	Type      OSType    `json:"type"`
-	SMBios    *SMBios   `json:"smBIOS,omitempty"`
-	BootOrder []Boot    `json:"bootOrder"`
```

Comment:
It is obsolete & very crude. The recommended way todo boot ordering is to set the boot index on individual devices.